docs/courseadministration/addcontent/quiz_import.md

```diff
@@ -0,0 +1,63 @@
+![HPI Logo](../../img/HPI_Logo.png)
+
+# Quiz Import from Google Spreadsheets
+
+You can import the quiz structure from Google Spreadsheets.
+All you have to do is the following:
+
+### 1. Make a copy of this template
+We can provide you with a quiz template that you can copy
+:::image type="content" source="../../img/courseadministration/quizmanagement/quiz_template.png" alt-text="How our quiz template looks like":::
```

Comment:
Could you have a look at these image tags here, please? Either use markdown syntax or HTML syntax here, this syntax might not work correctly.

Reply:
yep, it's fixed